contracts/reflect/src/contract.rs

```diff
@@ -97,6 +97,7 @@ pub fn query<S: Storage, A: Api, Q: Querier>(
         QueryMsg::Owner {} => to_binary(&query_owner(deps)?),
         QueryMsg::Capitalized { text } => to_binary(&query_capitalized(deps, text)?),
         QueryMsg::Chain { request } => to_binary(&query_chain(deps, &request)?),
+        QueryMsg::Raw { contract, key } => to_binary(&query_raw(deps, contract, key)?),
```

Comment:
Isn't that the same query that is implemented in `query_chain`?

Reply:
It is, but I test the query_raw helper and differentiate between None and Some(x) to make sure this really works as proposed.

Since we cannot unit test anything without wasmd, I just added this as a double-check

Reply:
The interface is nicer for testing, sure.

Do we need to want the ultra generic `QueryMsg::Chain` then?

Reply:
Yeah, for all the other queries I want to test - there are a lot of staking queries checked this way.
It was just the untestable custom handler for wasm raw and the empty data response I want to verify with this (as it caused a bit of a headache, let's triple check)